### PR TITLE
Range slider

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created range slider component.
Screen Shot 2022-02-19 at 8 46 10 PM
Screen Shot 2022-02-19 at 8 46 24 PM

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
Grid is good , looks good to merge

File: css/dropdown-select.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.custom-select select {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  -ms-appearance: none;
+  appearance: none;
+  border: 0 !important;
```
Comment (ritikaawasthy) [inline css/dropdown-select.css:6]: avoid using !important

File: css/dropdown-select.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.custom-select {
+  display: flex;
+  position: relative;
+  width: 20rem;
+  height: 3em;
+  overflow: hidden;
+  line-height: 3;
+  border-radius: 0.25rem;
```
Comment (ritikaawasthy) [inline css/dropdown-select.css:24]: use px for small values

**Review by ritikaawasthy — APPROVED** [review]
some minor issues otherwise looks great

**Pull request merged**